packages/gatsby-theme-api-docs/src/components/resource/method/url.js

```diff
@@ -47,27 +41,25 @@ const SlashWrappingPath = (props) => {
 
 const Url = ({ method, methodColor, uris }) => {
   return (
-    <>
-      <Container>
-        <Type
-          css={css`
-            background-color: ${methodColor};
-          `}
-        >
-          {method}
-        </Type>
-        <p>
-          <BasePath>
-            <SlashWrappingPath>
-              {uris.baseUri.replace(/^(https:\/\/)/, '')}
-            </SlashWrappingPath>
-          </BasePath>
-          <ResourceUriPath>
-            <SlashWrappingPath>{uris.resourcePathUri}</SlashWrappingPath>
-          </ResourceUriPath>
-        </p>
-      </Container>
-    </>
+    <SpacingsInline scale="xs">
+      <Type
+        css={css`
+          background-color: ${methodColor};
+        `}
+      >
+        {method}
+      </Type>
+      <SpacingsStack scale="xs">
+        <BasePath>
+          <SlashWrappingPath>
+            {uris.baseUri.replace(/^(https:\/\/)/, '')}
```

Comment:
```suggestion
            {uris.baseUri}
```
that was omitted in the design, but one reason to give the hostname always its own line was to have the proper URL including protocol again